src/Microsoft.Tye.Core/Microsoft.Tye.Core.csproj

```diff
@@ -32,6 +32,12 @@
     <Content Include="Templates\**" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="CoreStrings.resx" EmitFormatMethods="true">
+      <Generator></Generator>
```

Comment:
Isn't this supposed to be set to something?

Reply:
Directory.Build.targets sets stuff and it's filled in with arcade

Reply:
OK. I'm just not sure why we need the attribute and for it to be empty.

Reply:
Agree it’s triggering me

Reply:
I see now, let me try and remove it.